src/install/config/java.yaml

```
@@ -17,6 +17,7 @@ appInfo:
       - value: 'applicationServer'
         displayName: 'on an application server'
         recommendedGuided: true
+        icon: 'fe-thumbsup'
```

```suggestion
        icon: 'fe-server'
```

just because we are likely going to use that